JSONLD/consortia/dlr-bira.json

```diff
@@ -0,0 +1,18 @@
+{
+    "@id": "mip-cmor-tables/consortia/dlr-bira",
+    "@type": "cmip:consortium",
+    "consortium:cmip_acronym": "DLR-BIRA",
+    "consortium:description": "DLR-BIRA",
+    "consortium:url": "",
+    "consortium:changes": "",
+    "consortium:members": {
+        "@set": [
+            {
+                "@id": "Deutsches Zentrum fur Luft- und Raumfahrt"
+            },
+            {
+                "@id": "Royal Belgian Institute for Space Aeronomy"
```

Comment:
This follows a different format all descriptive strings rather than a path and then a descriptive string - compared to JSONLD/consortia/csiro-arccss.json

Reply:
This follows the text string with no acronym format. Also known as missing data.

Reply:
Moved to incomplete.